languages/java/exercises/concept/constructors/.meta/src/reference/java/RemoteControlCar.java

```diff
@@ -0,0 +1,46 @@
+class RemoteControlCar {
+    private final int speed;
```

Comment:
Are students familiar with the `final` modifier? The [docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file) have the following section on the example implementation files:

> The implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on).

Reply:
The final could be introduced at a later stage so I removed it. 
I noticed that I almost automatically adds final without thinking.